src/assets/schemas/config.schema.json

```diff
@@ -473,6 +474,80 @@
         }
       }
     },
+    "pagination": {
+      "additionalProperties": false,
+      "required": [
+        "source",
+        "type",
+        "params"
+      ],
+      "properties": {
+        "source": {
+          "enum": [
+            "query"
+          ],
+          "description": ""
+        },
+        "type": {
+          "enum": [
+            "lazy-loading"
+          ],
+          "description": ""
+        },
+        "params": {
+          "$ref": "paginationParams"
+        },
+        "fields": {
+          "$ref": "paginationFields"
+        }
+      }
+    },
+    "paginationParams": {
+      "additionalProperties": false,
+      "required": [
+        "page"
+      ],
+      "properties": {
+        "page": {
+          "$ref": "queryParamConfig"
+        },
+        "limit": {
+          "$ref": "queryParamConfig"
+        },
+        "sortBy": {
+          "$ref": "queryParamConfig"
+        },
+        "descending": {
+          "$ref": "queryParamConfig"
+        }
+      }
+    },
+    "paginationFields": {
+      "additionalProperties": false,
+      "properties": {
+        "page": {
```

Comment:
Yeah this is a bit ambiguous and it was in my mind too. Basically the reason is because the API that I use to do tests returns which page its returning. So the `page` information becomes an input (query param) and an output (body field). In that case, it is useless as an output honestly, I just use it to check if everything is coherent. 

But searching a bit about pagination methods in APIs ([like here](https://nordicapis.com/everything-you-need-to-know-about-api-pagination/)) I realized that there are other methods that the one we are implementing support for, and some give you "the next page" as an output, like a bookmark. Some give you the full URLs of the previous page, the next page, the first page and the last page.

Anyway, I think I'll remove it for this first implementation, but the next step is definitely going to be to implement support for other API pagination systems.

Reply:
Sounds good.

What about having in `paginationFields` another 2 fields, `nextPage` and `previousPage`, which could be either a URL, or a number?

Reply:
Next PR maybe? To go gradually?